flows/microstrategy/mstro_to_s3.py

```diff
@@ -0,0 +1,144 @@
+# Standard Library imports
+import datetime
+from io import StringIO
+import os
+
+# Prefect imports
+from prefect import task, Flow, Parameter, unmapped
+from prefect.engine.state import Failed
+from prefect.schedules import IntervalSchedule
+from prefect.run_configs import UniversalRun, DockerRun
+from prefect.backend import set_key_value, get_key_value
+from prefect.storage import Docker, GitHub
+from prefect.utilities.notifications import slack_notifier
+
+
+# Related third party imports
+import boto3
+from mstrio.connection import Connection
+from mstrio.project_objects import Report
+import pandas as pd
+
+handler = slack_notifier(only_states=[Failed])
+
+environment_variables = get_key_value(key=f"atd_microstrategy")
+
+ENV = "test"
+
+# Microstrategy Credentials
+PROJECT_ID = "6B64D80C11E1AFEA001000805B2705A0"
+BASE_URL = "https://mstrprod-library.austintexas.gov/MicroStrategyLibrary/api/"
+MSTRO_USERNAME = environment_variables["MSTRO_USERNAME"]
+MSTRO_PASSWORD = environment_variables["MSTRO_PASSWORD"]
+
+# 2D list of default reports to be retrived
+## [File Name for S3, Report ID]
+# To find report ID, go to the report in Microstrategy then:
+## Go to Tools > Report Details Page or Document Details Page.
+## Click Show Advanced Details button at the bottom
+# Report name should be unique as it is the file name in the S3 bucket
+REPORTS = [["2020 Bond Expenses Obligated", "85A9E0A049F06D98AF1CF3BE8CDA9394"]]
+
+## AWS Credentials
+AWS_ACCESS_ID = environment_variables["AWS_ACCESS_ID"]
+AWS_PASS = environment_variables["AWS_PASS"]
+BUCKET_NAME = environment_variables["BUCKET"]
```

Comment:
might be worth adding a copy of the env vars object you've got stored in prefect to 1Pass